src/mincost.jl

```diff
@@ -1,18 +1,22 @@
 """
-    mincost_flow(graph, capacity, demand, cost, solver, [, source][, sink])
+    mincost_flow(graph, node_demand, edge_capacity, edge_cost, solver,<keyword argument>)
 
-Find a flow satisfying the `demand` and `capacity` constraints for each edge
-while minimizing the `sum(cost.*flow)`.
+Find a flow satisfying the `node_demand` at each node and `edge_capacity` constraints for each edge
+while minimizing the `sum(edge_cost.*flow)`.
 
-- If `source` and `sink` are specified, they are allowed a net flow production,
-consumption respectively. All other nodes must respect the flow conservation
-property.
+-`node_demand` is a vector of nodal values, which should be positive for sources and
+ negative at sink nodes, any node with zero demand is a transport node
 
-- The problem can be seen as a linear programming problem and uses a LP 
+- The problem can be seen as a linear programming problem and uses a LP
 solver under the hood. We use Clp in the examples and tests.
 
 Returns a flow matrix, flow[i,j] corresponds to the flow on the (i,j) arc.
-
+# Arguments
+- `edge_demand::AbstractMatrix`: demand a minimum flow for a given edge.
```

Comment:
In the argument lists, `demand` was changed from Matrix to Vector, corresponding documentation changes should be required

Reply:
This is the documentation for the demand for a given edge, which is a matrix essentially equivalent to the old demand argument.   I think that the confusion was that in the doc, I called demand node_demand to add clarity.    Should I change the name in the actual code?